tasks/WhereIsThis.tex

```diff
@@ -1,16 +1,16 @@
 \section{Where is This? [Party Host]}
 \label{test:where-is-this}
-The robot has to explain to people where they can find places in the arena (e.g.~\emph{Where is the TV?}). The robot has either guide the operator there, or tell the operator how to get there (e.g.~\emph{It's at your right} or \emph{Turn around and go towards\dots}).
+The robot has to explain and show people where they can find places in the arena (e.g.~\emph{Where is the TV?}). The robot has to tell the operator how to get there (in a summarized fashion) and then physically take the operator there in a type of tour guide.
 
 
 \subsection*{Main Goal}
-Give accurate directions or guide at least 3 people.
+Give accurate directions and guide at least 3 people.
 
 \noindent\textbf{Reward:} 300pts (100pts per direction/guidance).
 
 \subsection*{Bonus rewards}
 \begin{enumerate}[nosep]
-	\item Give accurate directions or guide 3 more people 600pts (200pts each).
+	\item Give accurate directions and guide 3 more people 600pts (200pts each).
 	\item Provide new instructions based on the previous ones 300pts (100pts each).
 	\item Provide directions to a naive operator 300pts (50pts each).
 	\item Provide audio recording and transcript of each interaction 600pts (100pts each).
```

Comment:
This item is being discussed in the thread and not part of this PR diff. Did anyone do it in German Open?

I think the balance of points could lower the providing the transcript of each interaction to 50 pts each.

It seems like a lot to double the score of each direction/guidance when the effort and difficulty to record the data is relatively low, and it's a one time incurred effort. Whereas the behaviour of the recognition system is where we want to focus the points.

Reply:
Provided transcripts are accepted only if the robot solved the task, so no point farming. Besides, that's the only way to build an @Home training corpora.